linux-test/src/main/scala/org/apache/gearpump/test/linux/Util.scala

```diff
@@ -22,20 +22,13 @@ object Util extends App {
   lazy val httpsProxy = "https_proxy" -> "http://child-prc.intel.com:913"
   lazy val address = "https://github.com/intel-hadoop/gearpump.git"
   lazy val build = "sbt clean assembly packArchive"
-  lazy val sourceRoot = "/root/gearpump-yarn-test/gearpump"
+  lazy val sourceRoot = "/root/gearpump-test/gearpump"
```

Comment:
`/root/gearpump-test` means to run tests under root account. Better create a random temporary directory for testing.